app/src/bcsc-theme/features/settings/Settings.tsx

```diff
@@ -49,6 +50,19 @@ const Settings: React.FC = () => {
     logger.info('TODO: Settings action pressed')
   }
 
+  const onPressTermsOfUse = async () => {
+    try {
+      const supported = await Linking.canOpenURL(TERMS_OF_USE_URL)
+      if (supported) {
+        await Linking.openURL(TERMS_OF_USE_URL)
+      } else {
+        logger.error('Unable to open Terms of Use URL')
+      }
```

Comment:
question: do we need to call `canOpenURL` and `openURL` or can we just call `openURL`?

Reply:
Good point, simplified